### PR TITLE
adding support for wifi pineapple

### PR DESCRIPTION
**Pull request by tgalyean**
DD-WRT on the pineapple's dont have iproute2 installed.. added an if statement to utilize /proc to identify the default route if ip doesnt exist.. This should work on rpm, deb, and opkg based OS's..

### CODE REVIEW

File: wifiphisher.py [hunk 1]
```diff
@@ -362,7 +383,8 @@ def copy_AP():
     return channel, essid, mac
 
 def start_ap(mon_iface, channel, essid, args):
-    print '['+T+'*'+W+'] Starting the fake access point...'
+    print '['+T+'*'+W+'] Starting the fake access point on ' + mon_iface + '...'
+    time.sleep(2)
```
Comment (sophron) [inline wifiphisher.py]: What is this for?

File: wifiphisher.py [hunk 2]
```diff
@@ -775,6 +797,7 @@ def get_hostapd():
     # Main loop.
     try:
         while 1:
+            time.sleep(5)
```
Comment (sophron) [inline wifiphisher.py]: Also, what is this for?

File: wifiphisher.py (hunk 1 above)
Comment (tgalyean) [inline wifiphisher.py]: So, I added the sleep here and mon_iface because the print statement already existed in the code but it was unusable because as soon as the print occurred the screen would clear so you never got to see the information this provided. Also, I added the var for the iface so that you could confirm which interface was being used for monitor mode.

File: wifiphisher.py (hunk 2 above)
Comment (tgalyean) [inline wifiphisher.py]: The rapid refresh rate of the os.system("clear") causes periodic terminal hangs which disrupts the output therefore making the main status screen unusable (Kali Linux)

Comment (sophron) [inline wifiphisher.py]: This may makes sense but it is not relevant to the PR's purpose. Do you mind making this a different PR?

Comment (tgalyean) [inline wifiphisher.py]: Yep makes sense.

**Comment by tgalyean** [comment]
Splitting code into separate PR's
